frontend/public/locales/en/dental-insurance-question.json

```
@@ -1,27 +1,23 @@
 {
-  "title": "Access to dental insurance",
-  "legend": "Do you have access to dental insurance or coverage through one of the following four situations?",
-  "list-item1": "Your employment benefits or a family member’s employment benefits, including health and wellness accounts",
-  "list-item2": "Your pension benefits or a family member’s pension benefits",
-  "list-item3": "Purchased by you or a family member or through a group plan from an insurance or benefits company",
-  "list-item4": "Through a professional or student organization",
-  "content2": "You are still considered to have access to dental insurance or coverage if you choose to opt out of benefits provided through your employer/pension (see exception)  benefits or through a professional or student organization. As such, you are not eligible for CDCP.",
-  "option1": "I have access to dental insurance.",
-  "option2": "I do not have access to dental insurance.",
+  "title": "Access to other dental insurance",
+  "legend": "Do you have access to private dental benefits through any of the following situations?",
+  "list-item1": "Your <strong>employer</strong> or a family member’s employer dental insurance plan",
+  "list-item2": "Your <strong>pension</strong> or a family member’s pension dental insurance plan",
+  "list-item3": "You or your family member <strong>purchased dental insurance coverage</strong> from an insurance or benefits provider",
```

Use semantice name for the keys instead of key order. Use same pattern for the other keys.

```suggestion
  "list": {
    "employer-dental-insurance": "Your <strong>employer</strong> or a family member’s employer dental insurance plan",
    "pension-dental-insurance": "Your <strong>pension</strong> or a family member’s pension dental insurance plan",
    "purchased-insurance-coverage": "You or your family member <strong>purchased dental insurance coverage</strong> from an insurance or benefits provider"
  },
```